src/components/shared/text-field/TextField.tsx

```diff
@@ -32,10 +32,9 @@ const TextField = forwardRef<HTMLInputElement, TextFieldProps>(function TextFiel
   };
 
   // eslint-disable-next-line no-nested-ternary
-  const labelColor = hasError ? 'red' : focused ? 'primary' : undefined;
   return (
     <div>
-      {label && <Text typography="t6" display="inline-block" color={labelColor}>{label}</Text>}
+      {label && <Text typography="t6" display="inline-block" color="gray500" css={{ marginLeft: '15px' }}>{label}</Text>}
```

Comment:
const commonMarginLeft = '15px';

```suggestion
      {label && <Text typography="t6" display="inline-block" color="gray500"css={{ marginLeft: commonMarginLeft }}>{label}</Text>}
```
이부분이 중복되는데 변수처리해서 관리하는 것은 어떤가요??